rllib/utils/exploration/epsilon_greedy.py

```diff
@@ -139,7 +140,7 @@ def _get_torch_exploration_action(self, q_values, explore, timestep):
             # Mask out actions, whose Q-values are -inf, so that we don't
             # even consider them for exploration.
             random_valid_action_logits = torch.where(
-                q_values == -float("inf"),
+                q_values <= FLOAT_MIN,
```

Comment:
This is breaking tasks sometimes.

Reply:
Nice! Yeah, this could have screwed up sampling.